packages/headline/partials/pswp.hbs

```diff
@@ -12,10 +12,10 @@
             <div class="pswp__top-bar">
                 <div class="pswp__counter"></div>
 
-                <button class="pswp__button pswp__button--close" title="Close (Esc)"></button>
-                <button class="pswp__button pswp__button--share" title="Share"></button>
-                <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>
-                <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>
+                <button class="pswp__button pswp__button--close" title="{{t "Close (Esc)"}}"></button>
```

Comment:
In these kinds of places I'm not sure if it will work. Where would this show up on a page?